docs/site/Using-strong-error-handler.md

```diff
@@ -0,0 +1,343 @@
+---
+title: 'Using strong-error-handler'
+lang: en
+keywords: LoopBack 4, error handling
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Using-strong-error-handler.html
+summary:
+  Strong-error-handler is an error handler for use in both development and
+  production environments.
+---
+
+In LoopBack 4, the request handling process starts with the app's
+[sequence](https://loopback.io/doc/en/lb4/Sequence.html), a simple class with
+five injected helper methods in the constructor. It is the gatekeeper of all
+requests to the app. Errors are handled by one of the Sequence actions
+[`reject`](Sequence.md#handling-errors), which calls `strong-error-handler`
+package.
+
+# strong-error-handler
+
+This package is an error handler for use in both development (debug) and
+production environments.
+
+## Supported versions
+
+| Current | Long Term Support | End-of-Life |
+| :-----: | :---------------: | :---------: |
+|   4.x   |        3.x        |     2.x     |
```

Comment:
We usually format the LTS table differently: each major version is in a new row and we also show the EOL date for each version, see e.g. here:

https://github.com/strongloop/loopback#module-long-term-support-policy

I think there is no need to show 2.x version. The current 3.x version was published at [2018-06-11](https://github.com/strongloop/strong-error-handler/blob/master/CHANGES.md#2018-06-11-version-300), before LB4 GA. I would be very surprised if there was any LB4 user using 2.x version of strong-error handler.

Last but not least, AFAICT, the latest strong-error-handler version is 3.x. I am not sure where is 4.x coming from? See https://github.com/strongloop/strong-error-handler#supported-versions and https://github.com/strongloop/strong-error-handler/blob/master/package.json#L5